### PR TITLE
[SCRIPT][SQL] SCH AF (not helm) and Guttler (BST Relic)

### PR DESCRIPTION
**Pull request by nesstea**
Database controlled var-tracking and cleans up once they finish.

### CODE REVIEW

**Comment by nesstea** [comment]
Effayoo AppVeyor

**Comment by nesstea** [comment]
Amended last commit.

File: scripts/zones/Bastok_Markets_[S]/npcs/Loussaire.lua [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+-- onEventUpdate
+-----------------------------------
+
+function onEventUpdate(player,csid,option)
+    -- printf("CSID: %u",csid);
+    -- printf("RESULT: %u",option);
+
+    if (csid == 49 or csid == 53) then
+        -- Display Loafers
+        if     (option == 2) then
```
Comment (xdemolish) [inline scripts/zones/Bastok_Markets_[S]/npcs/Loussaire.lua]: ew

Comment (nesstea) [inline scripts/zones/Bastok_Markets_[S]/npcs/Loussaire.lua]: you're ew

Comment (TeoTwawki) [inline scripts/zones/Bastok_Markets_[S]/npcs/Loussaire.lua]: I second demo's "ew" :stuck_out_tongue:

File: scripts/zones/Pashhow_Marshlands_[S]/Zone.lua [hunk 2]
```diff
@@ -3,11 +3,12 @@
 -- Zone: Pashhow_Marshlands_[S] (90)
 --
 -----------------------------------
-package.loaded["scripts/zones/Pashhow_Marshlands_[S]/TextIDs"] = nil;
------------------------------------
 
-require("scripts/globals/settings");
+package.loaded["scripts/zones/Pashhow_Marshlands_[S]/TextIDs"] = nil;
```
Comment (TeoTwawki) [inline scripts/zones/Pashhow_Marshlands_[S]/Zone.lua]: AHEM. put that package nil back. It didn't need moved.

File: scripts/zones/Pashhow_Marshlands_[S]/npcs/Indescript_Markings.lua [hunk 3]
```diff
@@ -1,15 +1,18 @@
 -----------------------------------
 -- Area: Pashhow Marshlands [S]
 --  NPC: Indescript Markings
--- @pos -455 24 -364
------------------------------------
-package.loaded["scripts/zones/Pashhow_Marshlands_[S]/TextIDs"] = nil;
```
Comment (TeoTwawki) [inline scripts/zones/Pashhow_Marshlands_[S]/npcs/Indescript_Markings.lua]: Gonna effin slap ya.

File: scripts/globals/status.lua [hunk 4]
```diff
@@ -80,6 +80,8 @@ SUBEFFECT_HP_DRAIN          = 21;  -- 1-10101      43
 SUBEFFECT_MP_DRAIN          = 22;  -- This is correct animation
 SUBEFFECT_TP_DRAIN          = 22;  -- Verified this should look exactly like Aspir Samba.
 SUBEFFECT_HASTE             = 23;
+SUBEFFECT_CHOKE             = 24;
```
Comment (TeoTwawki) [inline scripts/globals/status.lua]: If you are certain the ID is correct, add the subeffect in source as well please https://github.com/DarkstarProject/darkstar/blob/c85204b161ba608cca351692d89bf26d51b811e7/src/map/entities/battleentity.h#L267

Note these are not arbitrary so you better have tested it locally.

File: scripts/globals/items/guttler.lua [hunk 5]
```diff
@@ -10,12 +10,13 @@ require("scripts/globals/magic");
 -- onAdditionalEffect Action
 -----------------------------------
 function onAdditionalEffect(player,target,damage)
+    
     local chance = 10;
-
+
```
Comment (TeoTwawki) [inline scripts/globals/items/guttler.lua]: PSA: Friends don't let friends add trailing tabs

File: scripts/globals/items/guttler.lua [hunk 6]
```diff
@@ -10,12 +10,13 @@ require("scripts/globals/magic");
...
 function onAdditionalEffect(player,target,damage)
+    
     local chance = 10;
-
+    
     if (math.random(0,99) >= chance or applyResistanceAddEffect(player,target,ELE_ICE,0) <= 0.5) then
         return 0,0,0;
-    else
-        target:addStatusEffect(EFFECT_CHOKE, 17, 0, 60);
-        return SUBEFFECT_CHOKE, MSGBASIC_ADD_EFFECT_STATUS, EFFECT_CHOKE;
```
Comment (TeoTwawki) [inline scripts/globals/items/guttler.lua]: Why did you move this?

Comment (xdemolish) [inline scripts/globals/items/guttler.lua]: i actually prefer it this way

Comment (TeoTwawki) [inline scripts/globals/items/guttler.lua]: That makes no sense. Either the condition is triggered or not, there is no 3rd possibility. All the other scripts use the previous format. Its a change with zero purpose.

File: scripts/zones/Pashhow_Marshlands_[S]/Zone.lua (hunk 2 above)
Comment (nesstea) [inline scripts/zones/Pashhow_Marshlands_[S]/Zone.lua]: LOL <3

File: sql/npc_list.sql [hunk 7]
```diff
@@ -9134,8 +9134,8 @@ INSERT INTO `npc_list` VALUES (17175333,'Moogle','Moogle',0,0.000,0.000,0.000,0,
 INSERT INTO `npc_list` VALUES (17175334,'Moogle','Moogle',0,0.000,0.000,0.000,0,40,40,0,0,0,2,4194307,0x0000520000000000000000000000000000000000,0,'WOTG');
 INSERT INTO `npc_list` VALUES (17175335,'Moogle','Moogle',0,0.000,0.000,0.000,0,40,40,0,0,0,2,4194307,0x0000520000000000000000000000000000000000,0,'WOTG');
 INSERT INTO `npc_list` VALUES (17175336,'Mithranbivouac','Moogle',203,386.661,-16.351,58.756,32769,40,40,0,16,112,0,3,0x0000340000000000000000000000000000000000,2,'WOTG');
-INSERT INTO `npc_list` VALUES (17175337,'Indescript_Markings','Mithran Bivouac',228,-358.657,-8.012,98.512,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
-INSERT INTO `npc_list` VALUES (17175338,'16','Indescript Markings',0,0.000,0.000,0.000,0,40,40,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
+INSERT INTO `npc_list` VALUES (17175337,'Mithran_Bivouac','Mithran Bivouac',228,-358.657,-8.012,98.512,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
```
Comment (teschnei) [inline sql/npc_list.sql]: we need to update the IDs before we can merge this.  it appears that starlightknight's github account is gone, so it's likely that one of us will have to do it

Comment (starlightknight) [inline sql/npc_list.sql]: @teschnei I'm around, not sure why my account would appear gone. I can look at ids within the next day or two

Comment (TeoTwawki) [inline sql/npc_list.sql]: @teschnei @starlightknight githubs been misbehaving lately so if there wasn't a typo with the name involded then its probably a temporary glitch. Yesterday I had `-2` contributions and the day before that I had infinity. Yes infinity. Before that a few people including myself had pull requests with either zero or infinity files changed.

Comment (teschnei) [inline sql/npc_list.sql]: okay, thanks @starlightknight. I tried using a mention in my earlier comment, but it didn't autocomplete, then I checked your commits in the darkstar-utils repo and they were unlinked to a github account /shrug

File: scripts/globals/items/guttler.lua (hunk 6 above)
Comment (nesstea) [inline scripts/globals/items/guttler.lua]: This works perfectly fine Teo. Less code, does the same thing.

File: scripts/globals/items/guttler.lua (hunk 5 above)
Comment (nesstea) [inline scripts/globals/items/guttler.lua]: It's 4 spaces... It gets ignored. Notepad++ autocomplete.

File: scripts/globals/status.lua (hunk 4 above)
Comment (nesstea) [inline scripts/globals/status.lua]: I have tested it.

File: scripts/globals/items/guttler.lua (hunk 6 above)
Comment (TeoTwawki) [inline scripts/globals/items/guttler.lua]: And deviating from every other existing script just because you felt like changing code that already worked. When I do it, I'm changing every instance trunk wide. It drives me insane to have your special unique snowflake staring back at me.

**Comment by teschnei** [comment]
now that we finally got npcids updated, resolve the conflicts from it and we can get back to this

**Comment by nesstea** [comment]
kk

**Comment by nesstea** [comment]
I am not sure how to resolve the conflicts. Should I close and resubmit?

**Comment by xdemolish** [comment]
pull from master it'll tell you what's causing conflicts
once you resolve conflicts, commit and rebase
then force push and leave a comment when it's dun

**Comment by nesstea** [comment]
It's showing conflicts in every file. I am going to redo this PR.

**Comment by xdemolish** [comment]
check your line endings, @teotwawki should be able to help you with resolving conflicts with those :p

**Comment by TeoTwawki** [comment]
`git config merge.renormalize true`
